source/php-libraries.txt

```diff
@@ -12,133 +12,83 @@ PHP Libraries, Frameworks, and Tools
-  <https://github.com/alcaeus/mongo-php-adapter>`_.
-
-- `Yamop <https://github.com/mawelous/yamop>`_ is yet another MongoDB ODM for
-  PHP. It works like the standard MongoDB PHP extension interface but returns
-  objects instead of arrays (as ODM). An integration with
-  `Laravel <https://github.com/mawelous/yamop-laravel>`_ is also available.
-
-Framework Integrations
-~~~~~~~~~~~~~~~~~~~~~~
+  - `DoctrineMongoDBBundle Symfony <https://github.com/doctrine/DoctrineMongoDBBundle>`__
+    This bundle integrates the Doctrine Object Document Mapper (ODM) into Symfony so
+    that you can store and retrieve objects from MongoDB.
 
-- Drupal
-
-  - `MongoDB integration for Drupal <https://www.drupal.org/project/mongodb>`_.
-    This is a collection of several modules which allow sites to store different
-    types of Drupal data in MongoDB. Support for the ``mongo`` extension exists
-    for Drupal 6, 7, and 8.
-
-- Kohana
-
-  - `MangoDB <https://github.com/Wouterrr/mangodb>`_: Mango is an ORM and
-    ActiveRecord-like library that takes full advantage of MongoDB's features.
-
-  - `MongoDB PHP ODM <https://github.com/colinmollenhour/mongodb-php-odm>`_ is a
-    simple but powerful set of wrappers for using MongoDB in PHP. It is designed
-    for use with Kohana 3 but should integrate easily with any PHP
-    application.
-
-- Yii 1.x
-
-  - `MongoYii <http://github.com/Sammaye/MongoYii/>`_ is ActiveRecord ORM for
-    Yii framework 1.x that supports MongoDB.
+- Yii2
 
-  - `Yii MongoDB Driver <https://github.com/fromYukki/Yii-MongoDB-Driver>`_ is a
-    MongoDB extension for Yii framework 1.x.
+  - `MongoDB Extension for Yii 2 <https://www.yiiframework.com/extension/yiisoft/yii2-mongodb>`__
+    is a MongoDB integration for the Yii 2 framework.
 
-Miscellaneous Projects
-~~~~~~~~~~~~~~~~~~~~~~
+Tools and Projects
+------------------
 
-- `PeclMongoPhpDoc <https://github.com/localgod/PeclMongoPhpDoc>`_ provides
-  skeleton classes for the ``mongo`` extension, which may be used to support
-  autocomplete and inline documentation for IDEs.
+- `PHP Cache <https://github.com/php-cache/mongodb-adapter/>`__ is a PSR-6 cache
+  implementation that uses MongoDB as a cache pool. This project is part of
+  the PHP Cache organization.
 
-- `PHPfastcache <https://github.com/PHPSocialNetwork/phpfastcache>`_ provides a simple,
+- `PHPfastcache <https://github.com/PHPSocialNetwork/phpfastcache>`__ is a
   high-performance backend cache system for MongoDB.
 
-- `XHGui <https://github.com/perftools/xhgui>`_, a web interface for the XHProf profiler
-  that stores profiling data in MongoDB.
+- `Enqueue <https://github.com/php-enqueue/mongodb>`__ is a production ready
+  messaging solution that uses MongoDB as the message queue broker. It provides
+  a common way for programs to create, send, read messages.
+
+- `XHGui <https://github.com/perftools/xhgui>`__ is a web interface for the
+  XHProf profiler which stores profiling data in MongoDB.
```

Comment:
I: Seems like this is a nonrestrictive clause, requiring a comma. (If it's restrictive, change to "that" and omit comma.)

```suggestion
  XHProf profiler, which stores profiling data in MongoDB.
```

Reply:
It provides additional detail to XMProf, so I think it is a nonrestrictive clause.